Sources/ConcurrencyHelpers/Protected.swift

```diff
@@ -0,0 +1,36 @@
+/// Thread safe access to simple variable protected by lock
+@propertyWrapper
+public final class Protected<T> {
+    var value: T
```

Comment:
`private` ?
Same for `lock`.

Reply:
If I understand correctly since It's property wrapper those values are hidden even for client within the same project, so no need to make them explicitly private.